app/routes/__layout/om-oss.tsx

```diff
@@ -1,11 +1,27 @@
+import { useLoaderData } from "@remix-run/react";
+import { json } from "@remix-run/server-runtime";
+
+import { BubbleGrid } from "~/components/bubbles/bubble-grid";
+import { fetchEmployeeImages } from "~/components/bubbles/capra-helper.server";
 import { CallToActionBox } from "~/components/call-to-action-box";
 import { ContentAndImageBox } from "~/components/content-and-image-box/content-and-image-box";
 import { TitleAndText } from "~/components/title-and-text";
 import { Todo } from "~/components/todo";
 import type { ValueProposition } from "~/components/value-wheel/value-wheel";
 import { ValueWheel } from "~/components/value-wheel/value-wheel";
+import { fetchImageAssets } from "~/utils/dataRetrieval";
+
+export const loader = async () => {
+  const [images, employeeImages] = await Promise.all([
+    fetchImageAssets([]),
```

Comment:
🗒️ Denne siden trenger å hente bilde resurser fra sanity, så bare tilrettela for det med en gang